tests/Auth/UserManagerClusterTest.cpp

```diff
@@ -32,7 +32,14 @@ namespace tests {
 namespace auth_info_test {
 
 namespace {
+#if defined(__APPLE__) && __clang_major__ == 13 && __clang_minor__ == 1
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunused-variable"
+#endif
 static char const* FailureOnLoadDB = "UserManager::performDBLookup";
+#if defined(__APPLE__) && __clang_major__ == 13 && __clang_minor__ == 1
+#pragma clang diagnostic pop
+#endif
```

Comment:
It's unused only in non-maintainer mode
```suggestion
[[maybe_unused]] static char const* FailureOnLoadDB = "UserManager::performDBLookup";
```